src/exampleTypes.js

```diff
@@ -148,9 +148,11 @@ export default F.stampKey('type', {
     shouldMergeResponse: node => node.infiniteScroll,
     mergeResponse(node, response, extend) {
       // extend but merge results arrays
-      extend(node.context, {
-        ...response.context,
-        results: [...node.context.results, ...response.context.results],
+      extend(node, {
+        context: {
+          ...response.context,
+          results: [...node.context.results, ...response.context.results],
+        },
```

Comment:
this part is important because we want the extend call to pick up on `context` as the key. For `watchNode` to work, we always need to extend on `node`